views/index.ejs

```diff
@@ -35,6 +35,38 @@
                 </div>
             </div>
         </section>
+        <section class="blog">
+            <div class="section_heading">
+                <h2>Blog</h2>
+                <div>Recent Posts</div>
+            </div>
+            <div id="post-container">
+                <div class="large-post odd-row">
+                    <img src="/assets/temp-blog-post.png">
+                    <h2>LOREM IPSUM DOLOR ></h2>
+                    <div class="subtitle">November 1, 2022</div>
+                </div>
+                <div class="small-post odd-row">
+                    <img src="/assets/temp-blog-post.png">
+                    <h2>TITLE TEXT 2: SUBTITLE</h2>
+                    <div class="subtitle">November 1, 2022</div>
+                </div>
+                <div class="small-post even-row">
+                    <img src="/assets/temp-blog-post.png">
+                    <h2>TITLE TEXT 2: SUBTITLE</h2>
+                    <div class="subtitle">November 1, 2022</div>
+                </div>
+                <div class="large-post even-row">
+                    <img src="/assets/temp-blog-post.png">
+                    <h2>LOREM IPSUM DOLOR ></h2>
+                    <div class="subtitle">November 1, 2022</div>
+                </div>
+            </div>
+
+            <div class="load-more">
+                <button>LOAD MORE</button>
```

Comment:
Brian brought up a good point last discussion, the load more button name is misleading if all we're doing is redirecting to the blog. Lets change this to 'SEE ALL'.